classes/class-wc-connect-api-client-live.php

```diff
@@ -17,20 +17,6 @@ class WC_Connect_API_Client_Live extends WC_Connect_API_Client {
 		protected function request( $method, $path, $body = array() ) {
 
 			// TODO - incorporate caching for repeated identical requests
-			if ( ! class_exists( '\Automattic\Jetpack\Connection\Manager' ) && ! class_exists( '\Automattic\Jetpack\Connection\Tokens' ) ) {
```

Comment:
These classes are always present because we bundle Jetpack Connection with WCS&T now.